packages/core/components/Modal/CodeSnippet/index.tsx

```diff
@@ -108,5 +108,5 @@ export default function CodeSnippet({ onDismiss }: ModalProps) {
         </>
     );
 
-    return <BaseModal body={body} onDismiss={onDismiss} title="Code snippet" />;
+    return <BaseModal body={body} onDismiss={onDismiss} title="Code snippet" isDraggable />;
```

Comment:
Kinda unclear why we ever want these to be draggable - should we just remove support for it?

Reply:
Removed in [2adc63c](https://github.com/AllenInstitute/biofile-finder/pull/475/commits/2adc63c65294879ecc03d07837f242a06ff3ea0e)